Debian/8/input/oval/oval_5.11/templates/Makefile

```diff
@@ -3,10 +3,16 @@ OUTPUT:=$(shell mkdir -p output)
 
 all:
 	${SHARED_DIR}/create_services_enabled.py services_enabled.csv
+	${SHARED_DIR}/create_services_disabled.py services_disabled.csv
 
-copy:
+copy: postbuild
 	cp output/*.xml ..
 
+# some shared oval need postbuild update.
+postbuild:
+#	ssh service is named ssh in Debian, not sshd, updating oval id.
+	sed -re 's:service_ssh_disabled:service_sshd_disabled:g' -i output/service_ssh_disabled.xml
```

Comment:
This should not be done here. It should be changed in the `<ocil>` and xccdf `description` instead. This breaks mappings elsewhere. `sshd` reference here is for the daemon that is running.